Cargo.toml

```diff
@@ -33,14 +33,11 @@ bytes = { version = "1", features=["serde"]}
 console-subscriber = { version = "0.1.6" , optional = true}
 drain = "0.1.1"
 futures = "0.3.12"
-futures-util = "0.3.21"
 gperftools = { version = "0.2.0", features = ["heap"], optional = true }
 hyper = { version = "0.14.18", features = ["full"] }
 hyper-boring = { version= "2.1.2" }
-lazy_static = "1.4.0"
```

Comment:
I think lazy_static is used in #159, please make sure not conflict

Reply:
Left comments on that PR to change to a new library. once_cell is the replacement for lazy_static